newsletter-august-2021.rst

```diff
@@ -0,0 +1,107 @@
+.. post:: August 9, 2021
+   :tags: newsletter, python
+   :author: Juan Luis
+   :location: MAD
+
+.. meta::
+   :description lang=en:
+      Company updates and new features from last month,
+      current focus, and upcoming features in August.
+
+Read the Docs newsletter - August 2021
+======================================
+
+Welcome to a new edition of our monthly newsletter, where we
+share the most relevant updates around Read the Docs,
+offer a summary of new features we shipped
+during the previous month,
+and share what we'll be focusing on in the near future.
```

Comment:
```suggestion
Welcome to the latest edition of our monthly newsletter, where we
share the most relevant updates around Read the Docs,
offer a summary of new features we shipped
during the previous month,
and share what we'll be focusing on in the near future.
```

I always read this as "welcome to our new newsletter" which confuses me every time.

Reply:
Hope to carry this out through the copy paste :D